### PR TITLE
Add big (1024MB) file ingestion scenario

### PR DESCRIPTION
**Pull request by dtitov**
```gherkin
  Scenario: R.1 User ingests big file encrypted with Crypt4GH using a correct key
    Given I have an account at Central EGA
    And I have correct private key
    And I connect to the LocalEGA inbox via SFTP using private key
    And I have a 1024 MB file encrypted with Crypt4GH using a LocalEGA's pubic key
    And I upload encrypted file to the LocalEGA inbox via SFTP
    And I have CEGA MQ username and password
    And I ingest file from the LocalEGA inbox
    When I retrieve ingestion information
    Then the ingestion status is "Completed"
```

The file size is configurable.

### CODE REVIEW

**Comment by silverdaz** [comment]
Looks good to me.

I have doubts that FileStatus.java is necessary (or maybe even that it might be a bad way to check if the file is ingested), but that'll do it. That can be adjusted later if necessary.

**Comment by silverdaz** [comment]
I can merge it. Do you want to wait for Juha to review this PR?

File: deployments/docker/tests/src/test/java/se/nbis/lega/cucumber/steps/Ingestion.java [hunk 1]
```diff
@@ -46,10 +50,10 @@ public Ingestion(Context context) {
         When("^I turn on the database", () -> utils.startContainer(utils.findContainer(utils.getProperty("images.name.db"),
                 utils.getProperty("container.name.db"))));
 
-        When("^I turn off the vault listener", () -> utils.stopContainer(utils.findContainer(utils.getProperty("images.name.mq"),
+        When("^I turn off RabbitMQ broker", () -> utils.stopContainer(utils.findContainer(utils.getProperty("images.name.mq"),
```
Comment (silverdaz) [inline deployments/docker/tests/src/test/java/se/nbis/lega/cucumber/steps/Ingestion.java]: Call it "the message broker". No need to settle for RabbitMQ specifically.

**Comment by dtitov** [comment]
I’m fine if you merge it, thanks.

**Review by silverdaz — APPROVED** [review]

File: deployments/docker/tests/src/test/java/se/nbis/lega/cucumber/steps/Ingestion.java (hunk 1 above)
Comment (dtitov) [reply deployments/docker/tests/src/test/java/se/nbis/lega/cucumber/steps/Ingestion.java]: Done.

**Pull request merged**